actors/README.md

```diff
@@ -44,10 +44,11 @@ be enough for most use cases.
 
 - Global actions can be added and removed from `Stage` with `+=` and `-=` operators.
 - Actions can be added and removed to individual `Actor` instances with `+=` and `-=` operators.
-- `Action.then` *infix* extension function allows to easily create action sequences with pleasant syntax.
-- `Action.parallelTo` *infix* extension function allows to easily create parallel actions with pleasant syntax.
-- `+` operator can be used to create action sequences (alternative syntax to `then`).
-- `+=` operator allows to add an action to an existing `SequenceAction`.
+- `Action.then` *infix* extension function allows easy creation of action sequences with pleasant syntax.
+- `Action.along` *infix* extension function allows easy creation of parallel actions with pleasant syntax.
+- `+` operator can be used to create action sequences (alternative non-mutating version of `then`).
+- `/` operator can be used combine actions in parallel (alternative non-mutating version of `along`).
+- `+=` operator allows to add an action to an existing `SequenceAction` or `ParallelAction`.
```

Comment:
If that's OK, you could add an additional section on when to use `then` vs `+` and `along` vs `/`.

Something along the lines of:
- Use operators for small amounts of actions (2-3) to prevent from creating too many objects. Prefer `then`/`along` for long chains of actions.
- Use operators to avoid mutating the sequences and parallel actions. If you are constructing actions in multiple steps and for any reason need a reference to intermediate step, use operators.
- Use `then`/`along` if you want to modify an existing sequence of parallel action without excessive creation of extra objects.